test/base/__init__.py

```diff
@@ -15,6 +15,12 @@
     log.addHandler(logging.StreamHandler())
 
 
+TEST_ROOT_URL = 'https://tc-tests.localhost'
+
+# rootUrl of a real deployment (that needs no pre-configuration)
+REAL_ROOT_URL = 'https://taskcluster.net'
```

Comment:
Other test helpers are attached to the TCTest class.  Let's keep it consistent and attach this to the TCTest class.